.github/workflows/potential-duplicates.yml

```diff
@@ -0,0 +1,29 @@
+name: Potential Duplicates
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: bubkoo/potential-duplicates@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          label: potential duplicate
+          exclude: |
+            icons
+            icon
+            logos
+            logo
+            adding
+            add
+            requesting
+            request
```

Comment:
I think that we can add some more words here:

- update
- updating
- need
- for
- remove
- create
- please
- and? (not sure about it)

Reply:
Good idea for a lot of them, thank you!

I voluntarily omit update and delete variations since I dont want those to be flagged with the original request.

Some common words are also already ignored (ex. `and`):
https://github.com/wow-actions/potential-duplicates/blob/master/src/dic.ts

I will add the other mention from your list